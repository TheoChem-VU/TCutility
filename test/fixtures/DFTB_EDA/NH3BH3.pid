# status: ready
# pid: 40578
# hostname: localhost
# username: 
# jobscript: 
# prolog: 
# epilog: 
# jobdir: $HOME/jobs
# fakejob: 0
# comparewithtarget: ok
# error: 0
# warning: 0
# runcmd: sh "$job" "NSCM=$options ; export NSCM"
# plamscmd: local:*,1
# batch: yes
# options: 
# killcmd: killall $pid
# jobstatuscmd: ps -e | grep -w $pid | grep -v grep
# sysstatuscmd: uptime
# pidfile: no
# label: Sequential
# logfile: logfile
# backup: 
# whenready: 
# scmjobid: 1692949463867-36503
# paused: 0
# remoteprefix: 
# jobtype: dftb
# mtime: 1692949760
# cloudinit: 
# cloudid: 
# copyback: 
# privatesshkeypath: 
# comments: 
# dependency: start /Users/yumanhordijk/ADF_DATA/dftb/EDA/NH3BH3.Acceptor.job
# dependency: start /Users/yumanhordijk/ADF_DATA/dftb/EDA/NH3BH3.Donor.job
