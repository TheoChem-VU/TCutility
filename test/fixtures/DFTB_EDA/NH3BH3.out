

 Parallel Execution: Process Information
 ==============================================================================
 Rank   Node Name                              NodeID   MyNodeRank  NodeMaster
    0   yumans-mbp.home                           0          0          0
    1   yumans-mbp.home                           0          1         -1
    2   yumans-mbp.home                           0          2         -1
    3   yumans-mbp.home                           0          3         -1
    4   yumans-mbp.home                           0          4         -1
    5   yumans-mbp.home                           0          5         -1
    6   yumans-mbp.home                           0          6         -1
    7   yumans-mbp.home                           0          7         -1
    8   yumans-mbp.home                           0          8         -1
    9   yumans-mbp.home                           0          9         -1
 ==============================================================================


May use up to 15625MB of RAM as shared memory on node 0

 *******************************************************************************
 *                                                                             *
 *  --------------------------------                                           *
 *   Amsterdam Modeling Suite (AMS)              2022.103                      *
 *  --------------------------------                                           *
 *                                               r104886 2022-06-17            *
 *                                                                             *
 *                                                                             *
 *                              =================                              *
 *                              |               |                              *
 *                              |     A M S     |                              *
 *                              |               |                              *
 *                              =================                              *
 *                                                                             *
 *                                                                             *
 *   Online information and documentation:  https://www.scm.com/support/       *
 *   E-mail:  support@scm.com   info@scm.com                                   *
 *                                                                             *
 *   Scientific publications using AMS results must be properly referenced     *
 *   See the User Manuals (or the web site) for recommended citations          *
 *   The terms and conditions of the End User License Agreement apply to       *
 *   the use of AMS, https://www.scm.com/license-terms/                        *
 *                                                                             *
 **********************  x86_64_darwin_intel / openmpi  ************************
 
 Licensed to: Prof Visscher, Prof Bickelhaupt, Prof Fonseca Guerra / Vrije Universiteit, Theoretische Chemie / Amsterdam / NETHERLANDS 2022-12-13 09:59:29
 SCM User ID: u21794
  
 AMS 2022.103  RunTime: Aug25-2023 09:49:18  ShM Nodes: 1  Procs: 10

 AMS jobname: ams
 AMS jobid  : 879464477

 Start   directory: /Users/yumanhordijk/ADF_DATA/dftb/EDA/tmp.yumanhordijk.40578.0.noindex/
 Results directory: /Users/yumanhordijk/ADF_DATA/dftb/EDA/NH3BH3.results/
 Scratch directory: /tmp/amstmp_ams_kid0.879464477/


 Communication costs MPI_COMM_WORLD:      2.570 usec per message,   0.0256 usec per 8-byte item
 Communication costs for intra-node:      2.214 usec per message,   0.0364 usec per 8-byte item

 RNG seed: -2009768965 -1691527481 -1198987668 1195871639 -2043752745 74139341 961924414 -294011729
       ****************************
       * SINGLE POINT CALCULATION *
       ****************************
 
--------
Geometry
--------
Formula: H6BN
Atoms
  Index Symbol   x (angstrom)   y (angstrom)   z (angstrom)
      1      N     0.00000000     0.00000000    -0.81474153       region=Donor
      2      B    -0.00000000    -0.00000000     0.83567034       region=Acceptor
      3      H     0.47608351    -0.82460084    -1.14410295       region=Donor
      4      H     0.47608351     0.82460084    -1.14410295       region=Donor
      5      H    -0.95216703     0.00000000    -1.14410295       region=Donor
      6      H    -0.58149793     1.00718395     1.13712667       region=Acceptor
      7      H    -0.58149793    -1.00718395     1.13712667       region=Acceptor
      8      H     1.16299585    -0.00000000     1.13712667       region=Acceptor
 
Total System Charge             0.00000
Bonding info available!
 
 
-------------------------
DFTB Engine configuration
-------------------------
DFTB Resources dir                     /Applications/AMS2022.103.app/Contents/Resources/amshome/atomicdata/DFTB/GFN1-xTB
 
SCC convergence enabled.
Maximum SCC cycles                       500
SCC charge convergence                   0.000000010000
SCC charge mixing                        0.150000000000
DIIS max dimension                       20
DIIS maximum coeff.                     10.000000000000
Adaptive SCC charge mixing             T
Adaptive SCC mixing strategy              3
Spin polarization                      F
Orbital-dependent SCC                  T
Orbital fill strategy                  Fermi
Fermi temperature (kelvin)                     300.0000
  ... kT (Hartree)                               0.0010
Use of symmetry                        T
Radial function extrapolation method   Auto

 Settings for Grimme D3 dispersion correction

      damping                        BJ
      s6                          1.000
      s8                          2.400
      a1                          0.630
      a2                          5.000

 Other (technical) parameters

      alpha                      14.000
      version                         4
 

---------------------
Periodic DFTB options
---------------------
 kpoints config
 grid method with params
Assume insulator                        F
Ewald tolerance                    1.00E-10
Ewald range factor                  2.00

 BZStruct config
 ---------------
 Enabled        T
 Automatic      T
 CalcFatBands   T
 useSymmetry    T
 deltaK          0.100000000000000     
 
Estimated upper bound for memory consumption:    0.065 MiB

Number of basis functions:           20
    Cycle:   1, MaxQDiff:   1.124721150628, PartialEnergy:      -7.9018486819, Used mix:   0.150 nvec:    0
    Cycle:   2, MaxQDiff:   1.039625587082, PartialEnergy:      -7.9039135527, Used mix:   0.150 nvec:    1
    Cycle:   3, MaxQDiff:   0.883987160608, PartialEnergy:      -7.9070971462, Used mix:   0.150 nvec:    1
    Cycle:   4, MaxQDiff:   0.061738779575, PartialEnergy:      -7.9124262037, Used mix:   0.150 nvec:    2
    Cycle:   5, MaxQDiff:   0.049367143646, PartialEnergy:      -7.9125752986, Used mix:   0.150 nvec:    2
    Cycle:   6, MaxQDiff:   0.005861501455, PartialEnergy:      -7.9128477067, Used mix:   0.150 nvec:    3
    Cycle:   7, MaxQDiff:   0.004440520572, PartialEnergy:      -7.9128488338, Used mix:   0.150 nvec:    2
    Cycle:   8, MaxQDiff:   0.000613415624, PartialEnergy:      -7.9128503697, Used mix:   0.150 nvec:    3
    Cycle:   9, MaxQDiff:   0.000492170197, PartialEnergy:      -7.9128503752, Used mix:   0.150 nvec:    3
    Cycle:  10, MaxQDiff:   0.000096317181, PartialEnergy:      -7.9128503797, Used mix:   0.150 nvec:    4
    Cycle:  11, MaxQDiff:   0.000078627919, PartialEnergy:      -7.9128503802, Used mix:   0.150 nvec:    4
    Cycle:  12, MaxQDiff:   0.000002614867, PartialEnergy:      -7.9128503808, Used mix:   0.150 nvec:    5
    Cycle:  13, MaxQDiff:   0.000002184263, PartialEnergy:      -7.9128503808, Used mix:   0.150 nvec:    5
    Cycle:  14, MaxQDiff:   0.000000000196, PartialEnergy:      -7.9128503808, Used mix:   0.150 nvec:    6
 Evaluation was successful



 =================
 Fragment orbitals
 =================
  
 Fragment 1, Formula: H3B_1, File: Acceptor.rkf
 Fragment 2, Formula: H3N_2, File: Donor.rkf

 Frag   FO  Energy(hartree)     Energy(eV)   Occ
 ------------------------------------------------
   1     1       -0.56769707        -15.448  2.00
         2       -0.46394545        -12.625  2.00
         3       -0.46394545        -12.625  2.00
         4       -0.31181129         -8.485  0.00
         5       -0.09004963         -2.450  0.00
         6       -0.05798294         -1.578  0.00
         7       -0.05798294         -1.578  0.00
         8        0.23675762          6.443  0.00
         9        0.28011621          7.622  0.00
        10        0.28011622          7.622  0.00
   2     1       -0.73722514        -20.061  2.00
         2       -0.53233829        -14.486  2.00
         3       -0.53233829        -14.486  2.00
         4       -0.40740727        -11.086  2.00
         5       -0.12905659         -3.512  0.00
         6       -0.04224483         -1.150  0.00
         7       -0.04224483         -1.150  0.00
         8        0.29142899          7.930  0.00
         9        0.29142899          7.930  0.00
        10        0.67883488         18.472  0.00



 =====================================================
 F O   P O P U L A T I O N S ,   M O   A N A L Y S I S
 =====================================================
  


 A Mulliken population analysis is performed. All populations refer to FOs.

 FO contributions (%) per orbital
 (multiplication by the orbital occupation yields the FO Gross Populations)

 Orb.:           2      3      4      5      6      7      8      9     10     11     12     13     14     15
 occup:        2.00   2.00   2.00   2.00   2.00   2.00   0.00   0.00   0.00   0.00   0.00   0.00   0.00   0.00
 Frag   FO     ----   ----   ----   ----   ----   ----   ----   ----   ----   ----   ----   ----   ----   ----
 ---------
   1     1:    0.00   0.00  37.91  58.45   0.00   0.00   0.20   0.00   0.00   0.26   0.00   0.00   2.67   0.00
         2:    0.00   1.16   0.00   0.00  98.30   0.00   0.00   0.00   0.35   0.00   0.02   0.00   0.00   0.10
         3:    1.16   0.00   0.00   0.00   0.00  98.30   0.00   0.35   0.00   0.00   0.00   0.02   0.00   0.00
         4:    0.00   0.00   5.87  15.52   0.00   0.00   0.76   0.00   0.00   6.18   0.00   0.00  66.48   0.00
         5:    0.00  -0.00   0.02   0.01   0.00   0.00  -0.03   0.00   0.00  90.16   0.00   0.00   9.95   0.00
         6:    0.00   0.01   0.00   0.00   0.00   0.00   0.00   0.00   2.25   0.00  97.69   0.00   0.00   0.04
         7:    0.01   0.00   0.00   0.00   0.00   0.00   0.00   2.25   0.00   0.00   0.00  97.69   0.00   0.00
         8:   -0.00   0.00   0.01   0.07   0.00   0.00   0.14  -0.00   0.00   0.09   0.00   0.00   1.72   0.00
         9:    0.12   0.00   0.00   0.00   0.00   0.05   0.00   0.03   0.00   0.00   0.00   0.03   0.00   0.00
        10:    0.00   0.12   0.00   0.00   0.05   0.00   0.00   0.00   0.03   0.00   0.03   0.00   0.00   3.23
   2     1:    0.00   0.00   0.34   0.00   0.00   0.00   0.02   0.00   0.00   0.09   0.00   0.00   0.37   0.00
         2:   98.66   0.00   0.00   0.00   0.00   1.15   0.00   0.00   0.00   0.00   0.00   0.02   0.00   0.00
         3:    0.00  98.66   0.00   0.00   1.15   0.00   0.00   0.00   0.00   0.00   0.02   0.00   0.00   0.07
         4:    0.00   0.00  56.05  24.81   0.00   0.00   0.06   0.00   0.00   2.25   0.00   0.00  15.69   0.00
         5:    0.00  -0.00  -0.24   1.15   0.00  -0.00  98.85   0.00   0.00   0.19   0.00  -0.00  -0.09   0.00
         6:   -0.00  -0.00   0.00   0.00   0.00   0.39   0.00  97.34   0.00   0.00   0.00   2.22   0.00   0.00
         7:   -0.00  -0.00  -0.00   0.00   0.39   0.00   0.00   0.00  97.34   0.00   2.22   0.00   0.00   0.00
         8:    0.00   0.04   0.00   0.00   0.10   0.00   0.00   0.00   0.02   0.00   0.02   0.00   0.00  96.56
         9:    0.04   0.00   0.00   0.00   0.00   0.10   0.00   0.02   0.00   0.00   0.00   0.02   0.00   0.00
        10:   -0.00   0.00   0.05  -0.02   0.00   0.00   0.01   0.00   0.00   0.78   0.00   0.00   3.20   0.00

 Orb.:          16
 occup:        0.00
 Frag   FO     ----
 ---------
   1     3:    0.10
         7:    0.04
         9:    3.23
   2     2:    0.07
         9:   96.56


 Summation over all MOs, multiplied by occupation: Total FO Gross Populations
 ============================================================================
 Fragment 1, H3B_1:
    1.93    1.99    1.99    0.44   -0.00    0.00    0.00    0.00    0.00    0.00
 Fragment 2, H3N_2:
    1.99    2.00    2.00    1.62    0.02    0.01    0.01    0.00    0.00    0.00



 List of all MOs with the most significant FO populations
 ========================================================

 Each percentage contribution in the table below corresponds to the indicated FO.

       E(eV)  Occ     MO       %       FO                E(eV)  Occ   Fragment
 -------------------------------------------------------------------------------

     -16.554  2.00     2     98.66%     2 A            -14.486  2.00     2 H3N_2
                              1.16%     3 A            -12.625  2.00     1 H3B_1
     -16.554  2.00     3     98.66%     3 A            -14.486  2.00     2 H3N_2
                              1.16%     2 A            -12.625  2.00     1 H3B_1
     -15.251  2.00     4     56.05%     4 A            -11.086  2.00     2 H3N_2
                             37.91%     1 A            -15.448  2.00     1 H3B_1
                              5.87%     4 A             -8.485  0.00     1 H3B_1
     -12.741  2.00     5     58.45%     1 A            -15.448  2.00     1 H3B_1
                             24.81%     4 A            -11.086  2.00     2 H3N_2
                             15.52%     4 A             -8.485  0.00     1 H3B_1
                              1.15%     5 A             -3.512  0.00     2 H3N_2
     -10.946  2.00     6     98.30%     2 A            -12.625  2.00     1 H3B_1
                              1.15%     3 A            -14.486  2.00     2 H3N_2
     -10.946  2.00     7     98.30%     3 A            -12.625  2.00     1 H3B_1
                              1.15%     2 A            -14.486  2.00     2 H3N_2
      -4.880  0.00     8     98.85%     5 A             -3.512  0.00     2 H3N_2
                              0.76%     4 A             -8.485  0.00     1 H3B_1
      -2.752  0.00     9     97.34%     6 A             -1.150  0.00     2 H3N_2
                              2.25%     7 A             -1.578  0.00     1 H3B_1
      -2.752  0.00    10     97.34%     7 A             -1.150  0.00     2 H3N_2
                              2.25%     6 A             -1.578  0.00     1 H3B_1
      -1.047  0.00    11     90.16%     5 A             -2.450  0.00     1 H3B_1
                              6.18%     4 A             -8.485  0.00     1 H3B_1
                              2.25%     4 A            -11.086  2.00     2 H3N_2
      -0.031  0.00    12     97.69%     6 A             -1.578  0.00     1 H3B_1
                              2.22%     7 A             -1.150  0.00     2 H3N_2
      -0.031  0.00    13     97.69%     7 A             -1.578  0.00     1 H3B_1
                              2.22%     6 A             -1.150  0.00     2 H3N_2
       1.399  0.00    14     66.48%     4 A             -8.485  0.00     1 H3B_1
                             15.69%     4 A            -11.086  2.00     2 H3N_2
                              9.95%     5 A             -2.450  0.00     1 H3B_1
                              3.20%    10 A             18.472  0.00     2 H3N_2
                              2.67%     1 A            -15.448  2.00     1 H3B_1
                              1.72%     8 A              6.443  0.00     1 H3B_1
       5.918  0.00    15     96.56%     8 A              7.930  0.00     2 H3N_2
                              3.23%    10 A              7.622  0.00     1 H3B_1
       5.918  0.00    16     96.56%     9 A              7.930  0.00     2 H3N_2
                              3.23%     9 A              7.622  0.00     1 H3B_1



 ===============================
 Site Energies Fragment orbitals
 ===============================
  
 The Site energy of a fragment orbital (FO) is defined here as the diagonal Fock matrix element
 in the Fock matrix in FO representation.
 Fragment 1, Formula: H3B_1, File: Acceptor.rkf
 Fragment 2, Formula: H3N_2, File: Donor.rkf

 Frag   FO    Energy(eV) Site Energy(eV)  Occ
 ------------------------------------------------
   1     1        -15.448        -13.869  2.00
         2        -12.625        -11.062  2.00
         3        -12.625        -11.062  2.00
         4         -8.485         -6.754  0.00
         5         -2.450         -1.001  0.00
         6         -1.578         -0.152  0.00
         7         -1.578         -0.152  0.00
         8          6.443          7.998  0.00
         9          7.622          9.226  0.00
        10          7.622          9.226  0.00
   2     1        -20.061        -22.254  2.00
         2        -14.486        -16.512  2.00
         3        -14.486        -16.512  2.00
         4        -11.086        -13.391  2.00
         5         -3.512         -5.176  0.00
         6         -1.150         -2.787  0.00
         7         -1.150         -2.787  0.00
         8          7.930          5.977  0.00
         9          7.930          5.977  0.00
        10         18.472         16.698  0.00


    =====================
     CALCULATION RESULTS
    =====================

Energy (hartree)            -7.82088458


--------------
Atomic charges
--------------
  Index   Atom    Charge
      1      N   -0.3146
      2      B    0.0333
      3      H    0.2232
      4      H    0.2232
      5      H    0.2232
      6      H   -0.1295
      7      H   -0.1295
      8      H   -0.1295


Dipole moment vector    (Debye):    -0.00000000    0.00000000   -4.43608394
              magnitude (Debye):     4.43608394

 
--------------------
Energy Decomposition
--------------------
Total Energy (hartree)                -7.82088458
Total Energy (eV)                   -212.81709753
Electronic Energy (hartree)           -7.93826672
Coulomb Energy (hartree)               0.02541634
Repulsion Energy (hartree)             0.09334726
Dispersion Energy (hartree)           -0.00138146
 
----------------
Mulliken Charges
----------------
  Index   Atom          Charge      Population
      1      N   -0.3145874994        5.314587
      2      B    0.0333416188        2.966658
      3      H    0.2232056316        0.776794
      4      H    0.2232056316        0.776794
      5      H    0.2232056312        0.776794
      6      H   -0.1294570047        1.129457
      7      H   -0.1294570047        1.129457
      8      H   -0.1294570044        1.129457
         Total   -0.0000000000       14.000000
 
----------------------
Mulliken Shell Charges
----------------------
  Index   Atom      L          Charge      Population
      1      N      0    0.5462243022        1.453776
      2      N      1   -0.8608118017        3.860812
      3      B      0    1.1214401095        0.878560
      4      B      1   -1.0880984907        2.088098
      5      H      0    0.2465278936        0.753472
      6      H      0   -0.0233222619        0.023322
      7      H      0    0.2465278936        0.753472
      8      H      0   -0.0233222619        0.023322
      9      H      0    0.2465278933        0.753472
     10      H      0   -0.0233222620        0.023322
     11      H      0   -0.1248456587        1.124846
     12      H      0   -0.0046113460        0.004611
     13      H      0   -0.1248456587        1.124846
     14      H      0   -0.0046113460        0.004611
     15      H      0   -0.1248456586        1.124846
     16      H      0   -0.0046113458        0.004611
                Total   -0.0000000000       14.000000
 
 
--------
Orbitals
--------
Index of HOMO                    7
Fermi Energy (hartree)           -0.30115760
Orbitals
   Index  Energy (hartree)       Energy (eV)        Occupation  Degeneracy
       1       -0.81903521      -22.28708205        2.00000000           1
       2       -0.60833584      -16.55366059        2.00000000           2
       3       -0.60833584      -16.55366057        2.00000000           2
       4       -0.56046817      -15.25111488        2.00000000           1
       5       -0.46821380      -12.74074577        2.00000000           1
       6       -0.40227392      -10.94643035        2.00000000           2
       7       -0.40227392      -10.94643033        2.00000000           2
       8       -0.17932877       -4.87978416        0.00000000           1
       9       -0.10112896       -2.75185900        0.00000000           2
      10       -0.10112896       -2.75185899        0.00000000           2
      11       -0.03847001       -1.04682217        0.00000000           1
      12       -0.00112805       -0.03069592        0.00000000           2
      13       -0.00112805       -0.03069591        0.00000000           2
      14        0.05141884        1.39917784        0.00000000           1
      15        0.21748906        5.91817847        0.00000000           2
      16        0.21748907        5.91817860        0.00000000           2
      17        0.30409289        8.27478852        0.00000000           1
      18        0.35788858        9.73864390        0.00000000           2
      19        0.35788859        9.73864407        0.00000000           2
      20        0.65219229       17.74705519        0.00000000           1
 

AMS application finished. Exiting.
NORMAL TERMINATION
     Total cpu time:       1.01
  Total system time:       0.77
 Total elapsed time:       2.06
